js/app.js

```diff
@@ -475,6 +475,7 @@ App.prototype.runAnalyzer = function() {
 		description: this.analyzerData.meta,
 		url: this.analyzerData.url,
 		title: this.analyzerData.metaTitle,
+		titleWidth: this.snippetPreview.data.titleWidth,
```

Comment:
Why not introduce a `getTitleWidth` function on the snippet preview instead of grabbing it from the internals of the snippet preview?